src/execution/error.rs

```diff
@@ -266,10 +261,39 @@ impl<'a> std::fmt::Display for Excerpt<'a> {
             "{}{}{}{}{}",
             " ".repeat(self.indent),
             " ".repeat(self.gutter_width()),
-            Colour::Blue.paint(" | "),
+            blue(" | "),
             " ".repeat(self.location.column),
-            Colour::Green.bold().paint("^")
+            green_bold("^")
         )?;
         Ok(())
     }
 }
+
+// coloring functions
+
+#[cfg(feature = "term-colors")]
+fn blue(str: &str) -> impl std::fmt::Display {
+    str.blue()
+}
+#[cfg(not(feature = "term-colors"))]
+fn blue<'a>(str: &'a str) -> impl std::fmt::Display + 'a {
```

Comment:
nit: Do you need the explicit lifetime bounds here?  Does it work to give this the same signature as the `term-colors` variant above?